source/includes/Person/person_attribute_type.md

```diff
@@ -57,38 +90,6 @@ GET /personattributetype?q=race&v=default
 
 ```
 
-```java
-
-OkHttpClient client = new OkHttpClient().newBuilder()
-  .build();
-Request request = new Request.Builder()
-  .url("/openmrs/ws/rest/v1/personattributetype?q=race&v=default")
-  .method("GET", null)
-  .addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
-  .addHeader("Cookie", "JSESSIONID=1A5193DBE052C38DC303BAD947A05A83")
-  .build();
-Response response = client.newCall(request).execute();
-
-```
-
-```javascript
-
-var myHeaders = new Headers();
-myHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
-myHeaders.append("Cookie", "JSESSIONID=1A5193DBE052C38DC303BAD947A05A83");
-
-var requestOptions = {
-  method: 'GET',
-  headers: myHeaders,
-  redirect: 'follow'
-};
-
-fetch("/openmrs/ws/rest/v1/personattributetype?q=race&v=default", requestOptions)
-  .then(response => response.text())
-  .then(result => console.log(result))
-  .catch(error => console.log('error', error));
-
-```
 
```

Comment:
this was the Issue the response has to be the last segment in the console and hence should come after all the laguage tab sections